CHANGELOG.rst

```diff
@@ -2,7 +2,8 @@
 
 Breaking changes:
   - Update components to use new Vivarium lookup table configuration
-  - Refactor Risk and RiskEffect components
+  - Refactor Risk component
+  - Implement new base RiskExposureDistribution class
```

Comment:
I don't think we have a new RiskExposureDistribution class (although there was a small change in it) so you can remove this. You can change the first one to  "refactor Risk component to create exposure in state table"

Reply:
@hussain-jafari We definitely do have a new RiskExposureDistribution class (it's a base class) - it may not necessarily need to be in the changelog though (I see you've already committed a change removing it).

Reply:
Oh my god yeah you're right. I was even thinking of saying _I_ didn't create a new RiskExposureDistribution class because I thought you might have been misinterpreting the changes I made in there. But yeah I remember Rajan making it. I don't think it's a breaking change, but can be included in major changes. I'll let you do that though!